src/main/java/gui/modal/DirectorShotCreationModalView.java

```diff
@@ -36,36 +36,37 @@
 
     // width and height of screen. 680 and 430 work very, very well.
     private static final int width = 680;
-    private static final int height = 430;
+    private static final int height = 460;
 
+
+    // three main colors used throughout window. Experiment a little!
+    private Color mainColor = Color.rgb(255, 172, 70); // main bright color
+    private Color secondaryColor = Color.rgb(255, 140, 0); // darker color
+    private Color tertiaryColor = Color.rgb(255, 235, 190); // lighter color
+
+    // variables for spacing
+    private int topAreaHeight = 70;
+    private int bottomAreaHeight = 60;
```

Comment:
Plan right now is to push most of these out of these classes somehow, so we can set application colors globally. But yeah, for now I'll make them static final.